R/derive_date_vars.R

```diff
@@ -427,20 +439,35 @@ compute_dtf <- function(dtc, dt) {
 #' compute_tmf(dtc = "2019-07-18T15:25", dtm = as.POSIXct("2019-07-18T15:25:00"))
 #' compute_tmf(dtc = "2019-07-18T15", dtm = as.POSIXct("2019-07-18T15:25:00"))
 #' compute_tmf(dtc = "2019-07-18", dtm = as.POSIXct("2019-07-18"))
-compute_tmf <- function(dtc, dtm) {
+compute_tmf <- function(dtc,
+                        dtm,
+                        ignore_seconds_flag = FALSE) {
+
   assert_that(is.character(dtc), is_date(dtm))
```

Comment:
Can you use `assert:character_vector()` instead and maybe add a new `assert_date_vector()` function in `assertions.R`? `assert_that()` error messages are rather verbose...

Reply:
Could this be split out into another issue - the `assert_date_vector()`?

Reply:
```
  assert_that(is_date(dtm))
  assert_character_vector(dtc)
  assert_logical_scalar(ignore_seconds_flag)
```
For now I just used the `assert_character_vector(dtc)` as you suggested

Reply:
@thomas-neitmann  actually - is_date() performs this check already 
```
 assert_that(is_date("2019-07-18T15:25:00"))
Error: Argument "2019-07-18T15:25:00" = 2019-07-18T15:25:00 is not a lubridate date.
```

Reply:
but maybe we should have a quick chat about it - if i do 

```
> is_date("2020-12-06")
[1] FALSE
```
then I don't get the Error message so I am happy to write a specific `assert_date_vector()` function, but can we move to a separate issue?  This `assert_that()` is used throughout this file so would need some more re-work.

Reply:
@thomas-neitmann Hey Thomas - can we make this into a separate issue.  It requires a bit of updating to the whole script and not just this section.

Reply:
Putting that into a spearate issue is fine with me 🙂